docs/supported_apis/older_pandas_compat.rst

```diff
@@ -1,37 +0,0 @@
-===================================
-Pandas backwards compatibility mode
```

Comment:
Maybe we should leave a mention of this mode, indicating the latest release, where is it available?

Reply:
Done.